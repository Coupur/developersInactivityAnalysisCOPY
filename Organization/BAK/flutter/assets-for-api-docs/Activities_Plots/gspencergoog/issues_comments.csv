id,date,creator_login
493274791,2019-05-17 00:22:47,gspencergoog
488354945,2019-05-01 17:43:01,gspencergoog
479157268,2019-04-02 19:20:10,gspencergoog
479157647,2019-04-02 19:21:05,gspencergoog
479163811,2019-04-02 19:35:40,gspencergoog
479174852,2019-04-02 20:01:33,gspencergoog
479176002,2019-04-02 20:04:15,gspencergoog
468398442,2019-02-28 19:10:25,gspencergoog
452388312,2019-01-08 17:43:53,gspencergoog
452049867,2019-01-07 19:20:39,gspencergoog
430400113,2018-10-16 21:03:51,gspencergoog
415592151,2018-08-23 22:32:53,gspencergoog
407795840,2018-07-25 15:29:59,gspencergoog
406373026,2018-07-19 18:33:20,gspencergoog
406388528,2018-07-19 19:30:14,gspencergoog
405765103,2018-07-17 23:59:13,gspencergoog
393016857,2018-05-30 03:13:17,gspencergoog
393210899,2018-05-30 15:41:48,gspencergoog
393337812,2018-05-30 22:17:35,gspencergoog
391948367,2018-05-25 05:47:17,gspencergoog
392140532,2018-05-25 18:19:14,gspencergoog
389255468,2018-05-15 17:50:50,gspencergoog
385187058,2018-04-28 16:10:11,gspencergoog
385122036,2018-04-28 00:00:39,gspencergoog
385143310,2018-04-28 05:53:20,gspencergoog
383738241,2018-04-23 22:06:15,gspencergoog
380524543,2018-04-11 17:01:27,gspencergoog
380629772,2018-04-11 23:45:17,gspencergoog
406727301,2018-07-20 21:07:32,gspencergoog
384467490,2018-04-25 23:42:42,gspencergoog
356788293,2018-01-11 00:52:10,gspencergoog
356786666,2018-01-11 00:42:45,gspencergoog
356441122,2018-01-09 22:54:24,gspencergoog
356734005,2018-01-10 20:56:18,gspencergoog
318241676,2017-07-27 02:37:28,gspencergoog
318248562,2017-07-27 03:28:14,gspencergoog
318421391,2017-07-27 16:52:19,gspencergoog
319737055,2017-08-02 17:10:31,gspencergoog
319829739,2017-08-02 23:53:04,gspencergoog
320123065,2017-08-04 00:15:48,gspencergoog
326343167,2017-08-31 16:01:45,gspencergoog
341841712,2017-11-03 22:22:26,gspencergoog
