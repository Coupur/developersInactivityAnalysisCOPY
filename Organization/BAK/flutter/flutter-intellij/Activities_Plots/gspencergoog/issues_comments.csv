id,date,creator_login
472133546,2019-03-12 18:47:43,gspencergoog
487116674,2019-04-26 16:22:00,gspencergoog
469734354,2019-03-05 15:55:57,gspencergoog
454086046,2019-01-14 17:17:45,gspencergoog
454992001,2019-01-17 00:08:04,gspencergoog
454993338,2019-01-17 00:14:59,gspencergoog
448480138,2018-12-19 06:05:32,gspencergoog
472121214,2019-03-12 18:15:55,gspencergoog
409386711,2018-07-31 22:19:11,gspencergoog
412602942,2018-08-13 17:42:39,gspencergoog
386116866,2018-05-02 20:51:17,gspencergoog
386122240,2018-05-02 21:09:47,gspencergoog
386123820,2018-05-02 21:15:28,gspencergoog
386162456,2018-05-03 00:35:14,gspencergoog
383241117,2018-04-20 22:44:50,gspencergoog
383251465,2018-04-21 00:07:37,gspencergoog
383259643,2018-04-21 02:08:10,gspencergoog
384717198,2018-04-26 17:07:56,gspencergoog
382825033,2018-04-19 17:53:30,gspencergoog
383160631,2018-04-20 17:06:09,gspencergoog
383199873,2018-04-20 19:36:06,gspencergoog
341531670,2017-11-02 19:25:08,gspencergoog
329330477,2017-09-13 23:59:59,gspencergoog
