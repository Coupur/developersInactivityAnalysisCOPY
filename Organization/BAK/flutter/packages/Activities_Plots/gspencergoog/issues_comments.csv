id,date,creator_login
513284918,2019-07-19 16:04:54,gspencergoog
457375983,2019-01-24 22:06:37,gspencergoog
434770808,2018-10-31 17:11:05,gspencergoog
431523720,2018-10-19 23:09:03,gspencergoog
457353549,2019-01-24 20:57:16,gspencergoog
427473054,2018-10-05 19:22:40,gspencergoog
414541415,2018-08-21 03:52:18,gspencergoog
415544884,2018-08-23 19:36:53,gspencergoog
398278847,2018-06-19 05:30:00,gspencergoog
403121484,2018-07-06 19:06:39,gspencergoog
408133153,2018-07-26 15:18:49,gspencergoog
408142111,2018-07-26 15:44:49,gspencergoog
397723694,2018-06-15 19:43:48,gspencergoog
397724006,2018-06-15 19:45:15,gspencergoog
398240294,2018-06-19 00:54:36,gspencergoog
