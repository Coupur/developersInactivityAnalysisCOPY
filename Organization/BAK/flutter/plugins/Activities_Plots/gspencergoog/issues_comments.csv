id,date,creator_login
435483015,2018-11-02 19:23:49,gspencergoog
427920055,2018-10-08 17:40:22,gspencergoog
430679509,2018-10-17 15:39:37,gspencergoog
414152905,2018-08-19 20:15:58,gspencergoog
414024153,2018-08-18 01:45:01,gspencergoog
414034953,2018-08-18 06:02:20,gspencergoog
414152972,2018-08-19 20:17:15,gspencergoog
414153091,2018-08-19 20:19:21,gspencergoog
414153131,2018-08-19 20:20:04,gspencergoog
416632822,2018-08-28 15:33:56,gspencergoog
369053289,2018-02-27 22:41:50,gspencergoog
