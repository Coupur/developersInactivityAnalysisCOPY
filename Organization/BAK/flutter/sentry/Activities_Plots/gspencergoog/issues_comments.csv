id,date,creator_login
369061132,2018-02-27 23:12:25,gspencergoog
