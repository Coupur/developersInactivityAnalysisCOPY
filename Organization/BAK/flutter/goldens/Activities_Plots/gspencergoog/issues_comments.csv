id,date,creator_login
410101196,2018-08-02 23:44:10,gspencergoog
